app/src/components/avatar_event_bus.rs

```diff
@@ -0,0 +1,55 @@
+use serde::{Deserialize, Serialize};
```

Comment:
Sorry, just noticed: the avatar event bus should be in the infra folder, not component